docs/io/ascii/write.rst

```diff
@@ -231,3 +231,181 @@ details.
   words, Reader classes can also write, but for historical reasons they are
   often called Reader classes.
 
+.. _cds_mrt_format:
+
+CDS/MRT Format
+----------
+
+Both `CDS <http://vizier.u-strasbg.fr/doc/catstd.htx>`_ and
+`Machine Readable Table (MRT) <https://journals.aas.org/mrt-standards/>`_ formats consist
+of a table description and the table data itself. MRT differs slightly from the CDS
+format in table description sections. CDS format includes more detailed description
+in the form of ``Abstract``, ``Notes``, ``References`` fields etc. and often has it in a
+separate file called ``ReadMe`` file. On the other hand, MRT format includes just the
+table ``Title``, ``Authors``, Table Caption and ``Notes`` and always has the ``ReadMe`` section
```

Comment:
Why do you set the Title in \`\`, but not Table caption?

Reply:
Because the MRT template has a field named _Title_ that needs to be manually filled. The Table Caption field is actually named _Table_, which can be ambiguous here. Also, the _Title_ refers to the paper/publication title in which this table is included.